example/timer/odp_timer_accuracy.c

```diff
@@ -1000,6 +1042,17 @@ static int run_test(void *arg)
 	stat->nsec_before_min = UINT64_MAX;
 	stat->nsec_after_min = UINT64_MAX;
 
+	if (test_global->opt.groups) {
+		odp_thrmask_zero(&mask);
+		odp_thrmask_set(&mask, tid);
+		group = test_global->group[tid - 1];
```

Comment:
It would be good to check tid < cpu_count, before using it. Just in case that thread IDs would have a problem.